content/SCALETutorials/Instances/_index.md

```diff
@@ -19,554 +19,633 @@ keywords:
 
-3. Select the path to the zvol created when setting up the VM on the **Zvol** dropdown list.
-4. Select the type of hard disk emulation from the **Mode** dropdown list.
-   Select **AHCI** for better software compatibility, or select **VirtIO** for better performance if the guest OS installed in the VM has support for VirtIO disk devices.
-5. (Optional) Specify the disk sector size in bytes in **Disk Sector Size**.
-   Leave set to **Default** or select either **512** or **4096** byte values from the dropdown list.
-   If not set, the sector size uses the ZFS volume values.
-6. Specify the boot order or priority level in **Device Order** to move this device up or down in the sequence.
-   The lower the number the higher the priority in the boot sequence.
-7. Click **Save**.
-8. Restart the VM.
+            Click **Create** to create the new volume.
 
-### Deleting a Disk Device
+         - Click **Import Zvols** on the **Volumes** screen opens the **Import Zvol** dialog.
 
-Deleting a device removes it from the list of available devices for the selected VM.
+            {{< trueimage src="/images/SCALE/Virtualization/InstanceImportZvol.png" alt="Import Zvol Dialog" id="Import Zvol Dialog" >}}
 
-To delete a VM device:
+            Enter a path in **Select Zvols** or browse to select an existing Zvol.
 
-1. Stop the VM if it is running, then click **Devices** to open the list of devices for the selected VM.
-2. Click on the <span class="material-icons">more_vert</span> icon at the right of the listed device you want to edit, then select **Delete**.
-   The **Delete Virtual Machine** dialog opens.
+            Select **Clone** to clone and promote a temporary snapshot of the zvol into a custom storage volume.
+            This option retains the original zvol while creating an identical copy as an instances volume.
+            Select **Move** to relocate the existing zvol to the ix-virt dataset as a volume.
 
-   {{< trueimage src="/images/SCALE/Virtualization/DeleteVirtualMachine.png" alt="Delete Virtual Machine" id="Delete Virtual Machine" >}}
+            Use this option to migrate a previously configured VM, such as after updating from TrueNAS 24.10.
+            See [Migrating Virtual Machines](https://www.truenas.com/docs/scale/25.04/gettingstarted/scalereleasenotes/#migrating-virtual-machines) from the 25.04 release notes for more information.
 
-3. Select **Delete zvol device** to confirm you want to delete the zvol device.
-   Select **Force Delete** if you want the system to force the deletion of the zvol device, even if other devices or services are using or affiliated with the zvol device.
-4. Click **Delete Device**.
+         - Click **Upload ISO** to open a file browser and select an <file>.iso</file> file from your client computer.
+            Locate your desired image file and then click **Open** to upload the file for use in instances.
 
-### Changing the Device Order
+2. Configure the **CPU & Memory** settings.
```

Comment:
don't use the section header as the noun in the action. you can make this 
```suggestion
2. Configure the CPU and memory settings.
```
or do something like this
```suggestion
2. Configure the CPU and memory settings in the  **CPU & Memory** section.
```